src/models/message_status.ts

```diff
@@ -35,9 +36,11 @@ export const MessageStatus = t.intersection([
     // eslint-disable-next-line sort-keys
     isArchived: withDefault(t.boolean, false)
   }),
-  // fiscalCode is optional due to retro-compatibility
   t.partial({
-    fiscalCode: FiscalCode
+    // fiscalCode is optional due to retro-compatibility
+    fiscalCode: FiscalCode,
+    // paymentStatus represents the status of a payment related to a message
+    paymentStatus: PaymentStatus
```

Comment:
Could be useful an additional payment status "UNKNOWN" on the model to use as default on old payament messages?
The payment info will not be migrated on old message_status, so using "NOT_PAYD" as default could not be right.

Reply:
So you suggest to move `paymentStatus` into required properties and manage a default?